receiver/otelarrowreceiver/config.go

```diff
@@ -54,3 +64,19 @@ func (cfg *ArrowConfig) Validate() error {
 	}
 	return nil
 }
+
+func (cfg *Config) Validate() error {
+	if err := cfg.GRPC.Validate(); err != nil {
+		return err
+	}
+	if err := cfg.Arrow.Validate(); err != nil {
+		return err
+	}
+	if cfg.Arrow.DeprecatedAdmissionLimitMiB != 0 {
+		cfg.Admission.RequestLimitMiB = cfg.Arrow.DeprecatedAdmissionLimitMiB
```

Comment:
This means that if a user specifies both the new and deprecated values, the deprecated value takes precedent. Is that what we want? Do we want to document this behavior?

Reply:
Thanks for pointing this out I changed the condition to be 
```
if cfg.Admission.RequestLimitMiB == 0 && cfg.Arrow.DeprecatedAdmissionLimitMiB != 0 {
```